src/lara-plugin/plugins/plugin-context.spec.ts

```diff
@@ -183,8 +183,8 @@ describe("Plugin runtime context helper", () => {
       interactiveStateUrl: "http://interactive.state.url",
       interactiveAvailable: true
     };
-    const runtimeContextWithEmbeddable = generateRuntimePluginContext(Object.assign({},
-      pluginContext, { wrappedEmbeddable }));
+    const runtimeContextWithEmbeddable = generateRuntimePluginContext({
+      ...pluginContext, wrappedEmbeddable});
```

Comment:
Note on this module:
Scott requested that I mark all code additions inside of the `lara-plugin` folder with the following:
`// ACTIVITY_PLAYER_CODE:`
And to comment out instead of removing any existing LARA code and mark the commented code with the following:
`// LARA_CODE`
I believe this is so we can more easily compare the activity player code with the original code in the LARA project and potentially unify the two code bases.